src/config.rs

```diff
@@ -1,3 +1,5 @@
+use nix::sys::signal::Signal;
+use nix::sys::signal::Signal::*;
```

Comment:
I'd prefer this import to be `nix::sys::signal{self, *};`

(sorry this is super nit-picky :sweat_smile::sweat_smile:)